scripts/deploy_teleporter.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+TELEPORTER_PATH=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")"
+    cd .. && pwd
+)
+
+if ! command -v forge &> /dev/null; then
+    echo "forge not found, installing"
+    $TELEPORTER_PATH/scripts/install_foundry.sh
```

Comment:
I'd prefer to have the script exit here rather than installing foundry automatically. That way the deployer can control what foundry installation they wish to use (the main advantage of our fork is stable CI, but it's by no means a requirement to using Teleporter ourside of CI). We should print that `install_foundry.sh` can be used to install our fork before exiting.

Reply:
Yep, good call.